package/MDAnalysis/lib/distances.py

```diff
@@ -206,6 +206,81 @@ def _check_lengths_match(*arrays):
     if not all( a.shape == ref for a in arrays):
         raise ValueError("Input arrays must all be same shape"
                          "Got {0}".format([a.shape for a in arrays]))
+      
+## NOT COMPLETE - NEED REFINEMENT
+def distance(atom1,atom2,box=None):      
```

Comment:
The call signature should be
```python
distance(a, b, box=None)
```
where `a` and `b` are coordinates (numpy arrays of shape `(3,)`). 

No functions in `MDAnalysis.lib` will make explicit reference to atoms or other "core data structures". Look at `distance_array()`.